createdisk-library.sh

```diff
@@ -114,16 +110,9 @@ function create_qemu_image {
 
 function create_bundle_qemu_image() {
   local libvirtDestDir="$1"
-  local VM_PREFIX="$2"
-  local VM_NAME="$3"
-
-  if [ "${BUNDLE_TYPE}" != "microshift" ]; then
-    create_qemu_image "$libvirtDestDir" "${VM_PREFIX}-base" "${VM_NAME}"
-    mv "${libvirtDestDir}/${VM_NAME}" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
-  else
-    create_qemu_image "$libvirtDestDir" "${VM_NAME}.qcow2" "microshift.qcow2"
-    mv "${libvirtDestDir}/microshift.qcow2" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
-  fi
+  local VM_NAME="$2"
+  create_qemu_image "$libvirtDestDir" "${VM_NAME}.qcow2" "${VM_NAME}"
```

Comment:
Are we still in the situation where the 3rd arg ("$VM_NAME") to `create_qemu_image` is an existing overlay file for openshift bundles, and a non-existent file for microshift? Or can `create_qemu_image` be simplified further? Looks like `create_bundle_qemu_image` and `create_qemu_image` could even become just one function?

Reply:
@cfergeau yes it can simplified further but I want to have this PR as simple as possible initially and then put another one around refactoring/simplification.

Reply:
I have to disagree with the PR being "as simple as possible", you already did 80% of the simplification work in create_bundle_qemu_image/create_qemu_image, and this makes it even harder than before to understand the `overlay` argument. 
If you want to keep things as simple as possible, I think this should be enough changes (disk image on the virt-install command line needs to be put in /var/libvirt/images/${SNC_PRODUCT_NAME}.qcow2)
```
diff --git a/createdisk-library.sh b/createdisk-library.sh
index 2b630bc..c80fd33 100755
--- a/createdisk-library.sh
+++ b/createdisk-library.sh
@@ -118,7 +118,7 @@ function create_bundle_qemu_image() {
   local VM_NAME="$3"
 
   if [ "${BUNDLE_TYPE}" != "microshift" ]; then
-    create_qemu_image "$libvirtDestDir" "${VM_PREFIX}-base" "${VM_NAME}"
+    create_qemu_image "$libvirtDestDir" "${SNC_PRODUCT_NAME}.qcow2" "${VM_NAME}"
     mv "${libvirtDestDir}/${VM_NAME}" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
   else
     create_qemu_image "$libvirtDestDir" "${VM_NAME}.qcow2" "microshift.qcow2"
```

Reply:
@cfergeau updated the PR and now have different commit to simplify it.